web_ui/src/routes/landing-page.layout.tsx

```diff
@@ -17,9 +18,9 @@ export const LandingPageLayout = (): JSX.Element => {
         <Grid
             height={'100vh'}
             maxHeight={'100vh'}
-            areas={['header header', 'banner banner', 'sidebar content']}
+            areas={['header header', 'banner banner', 'upgrade-banner upgrade-banner', 'sidebar content']}
```

Comment:
adding on all pages is ok but i dont know why we didnt do a bottom full width notification instead. It's standard and easier for us as well (1 single notification to maintain). wdyt?

Reply:
Yeah i know. This is by design and it's ok, i was just asking if maybe just a notification (i said full-width but it doesnt matter, any of our yellow notifications will do) could suffice